src/Resources/Table.php

```diff
@@ -680,7 +680,9 @@ protected static function fromMongoObjects($record)
                         $data = $data->toDateTime();
                         $data = ['$date' => $data->format($cfgFormat)];
                     } elseif ($data instanceof Binary) {
-                        $data = $data->getData();
+                        if ($data->getType() == 0) {
```

Comment:
@thekevinm What is this zero?
Also please always use strict comparison `===` 🙏

Reply:
This checks the binary subtype of the $data. In MongoDB, binary data can have different subtypes. The subtype 0 is the "generic" subtype, which is the default subtype and indicates that the binary data doesn't belong to a specific subtype.
If the binary data's subtype is 0, this line extracts the actual binary data from the Binary object. The getData() method retrieves the raw binary data as a string.
https://www.mongodb.com/docs/manual/reference/bson-types/#binary-data

Reply:
@thekevinm I added subtype enum class in case we need to use the other types in the future and updated the if condition you wrote 🙏